apps/music/index.html

```diff
@@ -116,7 +117,7 @@ <h1 id="title-text" data-l10n-id="music"></h1>
             <p data-l10n-id="music-paused-notification"></p>
           </section>
         </div>
-        <div id="player-seek">
+        <div id="player-seek" role="slider" aria-valuemin="0">
```

Comment:
Unfortunately we have to override the progress element here and use a role=slider instead. So the min/max/value attributes are copied to this element.